package.json

```diff
@@ -52,6 +62,18 @@
         "key": "ctrl+alt+h",
         "mac": "ctrl+alt+h",
         "when": "editorTextFocus"
+      },
+      {
+        "command": "extension.dash.syntax",
+        "key": "ctrl+shift+h",
+        "mac": "ctrl+shift+h",
+        "when": "editorTextFocus"
+      },
+      {
+        "command": "extension.dash.customWithSyntax",
+        "key": "shift+alt+h",
+        "mac": "shift+cmd+h",
```

Comment:
Control Cmd H works but I'm not sure what the equivalent is on windows. That being said, anything that doesn't conflict is fine with me :)